web/src/assets/locales/de.json

```
@@ -0,0 +1,217 @@
+          },
+          "internal": {
+            "internal": "Intern",
+            "desc": "Nur authentifizierte Benutzer der Woodpecker-Instanz können dieses Projekt sehen."
+          }
+        },
+
+        "timeout": {
+          "timeout": "Zeitlimit",
+          "minutes": "Minuten"
+        },
+
+        "cancel_prev": {
+          "cancel": "Breche ältere Pipelines ab",
+          "desc": "Aktiviere diese Option, um laufende Pipelines desselben Ereignisses und Kontexts abzubrechen, bevor die neue Pipeline startet."
+        }
+      },
+
+      "secrets": {
+        "secrets": "Geheimnisse",
+        "desc": "Geheimnisse können zur Laufzeit als Umgebungsvariablen an einzelne Pipelineschritte übergeben werden.",
+        "none": "Es gibt noch keine Geheimnisse.",
+        "add": "Geheimnis hinzufügen",
+        "save": "Geheimnis speichern",
+        "show": "Geheimnisse anzeigen",
+        "name": "Name",
+        "value": "Wert",
+        "deleted": "Geheimnis gelöscht",
+        "created": "Geheimnis erstellt",
+        "saved": "Geheimnis gespeichert",
+
+        "images": {
+          "images": "Verfügbar für folgende Images",
+          "desc": "Durch Kommata getrennte Liste der Images, für die dieses Geheimnis verfügbar ist; leer lassen, um alle Images zuzulassen"
+        },
+        "events": {
+          "events": "Verfügbar bei folgenden Ereignissen",
+          "pr_warning": "Sei bitte vorsichtig mit dieser Option, da ein bösartiger Akteur eine bösartigen Pull Request erstellen kann, der deine Geheimnisse preisgibt."
+        }
+      },
+
+      "registries": {
+        "registries": "Registries",
+        "creds": "Zugangsdaten für die Registry",
+        "desc": "Zugangsdaten für die Registry können hinzugefügt werden, um private Images für Ihre Pipeline zu verwenden.",
+        "show": "Registries anzeigen",
+        "add": "Registry hinzufügen",
+        "none": "Es gibt noch keine Zugangsdaten für die Registry.",
+        "save": "Registry speichern",
+        "created": "Registry-Zugangsdaten erstellt",
+        "saved": "Registry-Zugangsdaten gespeichert",
+        "deleted": "Registry-Zugangsdaten gelöscht",
+
+        "address": {
+          "address": "Adresse",
+          "placeholder": "Registratur Adresse (z. B. docker.io)"
+        }
+      },
+
+      "badge": {
+        "badge": "Abzeichen",
+        "url_branch": "URL für bestimmten Branch",
+        "markdown": "Markdown"
+      },
+
+      "actions": {
+        "actions": "Aktionen",
+
+        "repair": {
+          "repair": "Repository reparieren",
+          "success": "Repository repariert"
+        },
+        "disable": {
+          "disable": "Repository deaktivieren",
+          "success": "Repository deaktiviert"
+        },
+        "delete": {
+          "delete": "Repository löschen",
+          "confirm": "Alle Daten sind nach dieser Aktion verloren!!!\n\nMöchten Sie wirklich fortfahren?",
```

Looks like we used the du way in most other translations (which I would prefer over using Sie in general as well).

```suggestion
          "confirm": "Alle Daten sind nach dieser Aktion verloren!!!\n\nMöchtest du wirklich fortfahren?",
```